lib/seed/Seeder.js

```diff
@@ -76,11 +77,24 @@ class Seeder {
     return name;
   }
 
-  _getStubPath() {
-    return (
-      this.config.stub ||
-      path.join(__dirname, 'stub', this.config.extension + '.stub')
-    );
+  async _getStubPath() {
+    const packageDirectory = await pkgDir(process.cwd());
```

Comment:
does this need to happen on every method invocation? can't be moved outside of the function?

Reply:
maybe. Is it possible that a particular consuming package can install a copy of knex, and then a dependency of that package can use the same copy of knex?